lisa/util/__init__.py

```diff
@@ -657,3 +659,38 @@ def strip_strs(obj: Any, fields: List[str]) -> Any:
             value = value.strip() if isinstance(value, str) else value
             setattr(obj, field, value)
     return obj
+
+
+def check_till_timeout(
+    func: Callable[..., Any],
+    timeout_message: str,
+    timeout: int = 20,
+    waittime: int = 1,
+) -> None:
+    timer = create_timer()
+    while timer.elapsed(False) < timeout:
+        if func():
+            break
+        sleep(waittime)
+    if timer.elapsed() >= timeout:
+        raise LisaException(f"timeout: {timeout_message}")
+
+
+def getattr_with_check(actual_object: Any, attr_name: str = "") -> Any:
+    if hasattr(actual_object, attr_name):
+        return getattr(actual_object, attr_name)
+    raise LisaException(f"attr {attr_name} not existing in object {actual_object}")
+
+
+def method_with_arguments(
```

Comment:
method_with_arguments just an example, it doesn't need. Try to use the logic by lambda directly. If the logic is multiple lines, wrapper it to a function.